controllers/usersignup/usersignup_controller_test.go

```diff
@@ -3488,14 +3488,13 @@ func TestDeathBy100Signups(t *testing.T) {
 				},
 			})
 
-			for i := 2; i <= 100; i++ {
-				initObjs = append(initObjs, &toolchainv1alpha1.MasterUserRecord{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      fmt.Sprintf("%s-%d", testusername.replacedCompliantUsername, i),
-						Namespace: test.HostOperatorNs,
-						Labels:    map[string]string{toolchainv1alpha1.MasterUserRecordOwnerLabelKey: uuid.Must(uuid.NewV4()).String()},
-					},
-				})
+			for i := 2; i <= 100; i += 2 {
```

Comment:
I think without the context of this PR it might be a bit harder to understand this test right away so maybe we could add a short comment to explain it?
```suggestion
			// stagger the numbering for MURs and Spaces so that one of them will always be missing. eg. MURs will not be found on odd numbers and Spaces not found on even numbers until it makes 100 attempts
			for i := 2; i <= 100; i += 2 {
```

Reply:
thanks